conf/nginx.conf

```diff
@@ -1,3 +1,12 @@
+# Expires mappings
+map $sent_http_content_type $expires {
```

Comment:
From my tests (I hardcoded it on my server) this need to be on the same level as `server {`. Did you test it?

Reply:
> Did you test it?
No, I havn't.

Reply:
I double-checked: http://nginx.org/en/docs/http/ngx_http_map_module.html#map, `map` can only be defined in the `http` context, thus at the same level as the `server`.